src/Node/Buffer/Immutable.purs

```diff
@@ -100,7 +100,10 @@ foreign import concat :: Array ImmutableBuffer -> ImmutableBuffer
 
 -- | Concatenates a list of buffers, combining them into a new buffer of the
 -- | specified length.
-foreign import concat' :: Array ImmutableBuffer -> Int -> ImmutableBuffer
```

Comment:
Can you move the doc-comment to the version of this we are going to export please?

Reply:
Well spotted! I just did that.